generate_export_options.rb

```diff
@@ -98,6 +98,7 @@ def export_method(mobileprovision_content)
 
 export_options = {}
 export_options[:method] = method unless method.nil?
+export_options[:uploadBitcode] = false
 
```

Comment:
Could you please advise a PR-n00b how to handle my error of adding an extra commit to the PR?
Should I delete the PR entirely and create it from scratch?